### ISSUE TITLE
Color contrast does not meet minimum requirement: Dashboard navigation

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3076]
vpat.hykudemo.org
Version 7.1

### Descriptive summary
Color contrast for the language and login drop-down menus is not clear enough:
<img width="2018" height="1111" alt="Image" src="https://github.com/user-attachments/assets/fbee8fa5-a8b2-4b04-aaf0-36d2aeecd6cb" />

Siteimprove saw this issue across all Dashboard pages with this navigation. I am creating this single ticket because I think this is a single fix, not multiple. If I am not correct, these are the pages we have seen this issue on, but we assume all are affected:
- https://vpat.hykudemo.org/status?locale=en
- https://vpat.hykudemo.org/dashboard/my/collections?locale=en
- https://vpat.hykudemo.org/admin/account/edit?locale=en
- https://vpat.hykudemo.org/site/labels/edit?locale=en
- https://vpat.hykudemo.org/admin/features?locale=en
- https://vpat.hykudemo.org/admin/work_types/edit
- https://vpat.hykudemo.org/admin/roles_service?locale=en

### Rationale
Accessibility issue remediation

Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

To meet the WCAG contrast requirement, you should have a contrast ratio of 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

### Expected behavior
Color contrast should be at least 4.5: 1

### Actual behavior
Color contrast is 4.2:1

### Steps to reproduce the behavior
- Open the page
- Use the SiteImprove browser extension

### Related
